http-server-netty/src/main/java/io/micronaut/http/server/netty/handler/accesslog/element/AccessLog.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2017-2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.http.server.netty.handler.accesslog.element;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.slf4j.Logger;
+
+import io.micronaut.http.server.netty.handler.accesslog.element.AccessLogFormatParser.IndexedLogElement;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.socket.SocketChannel;
+import io.netty.handler.codec.http.HttpHeaders;
+
+/**
+ * An Access log instance.
+ *
+ * @author croudet
+ * @since 2.0
+ */
+public class AccessLog {
+    private List<IndexedLogElement> onRequestHeadersElements = new ArrayList<>();
+    private List<IndexedLogElement> onResponseHeadersElements = new ArrayList<>();
+    private List<IndexedLogElement> onResponseWriteElements = new ArrayList<>();
+    private List<IndexedLogElement> onLastResponseWriteElements = new ArrayList<>();
```

Comment:
It seems like these initializers are redundant and the fields can be made final since they are reassigned in the constructor?

Reply:
Fields made final. But they are not reassigned